R/modify_pd.R

```diff
@@ -16,7 +16,7 @@ indent_round <- function(pd, indent_by) {
     start <- stop <- 0
   }
   pd <- pd %>%
-    mutate(indent = ifelse(seq_len(nrow(pd)) %in% start:stop, indent_by, 0)) %>%
+    mutate(indent = indent + ifelse(seq_len(nrow(pd)) %in% start:stop, indent_by, 0)) %>%
```

Comment:
I think it's more coherent to initialise in `create_filler()` (as we also do with spacing and line break information. otherwise, you need a function like `indent_round()` to initialise. Say you don't use `indent_round()` for some reason, you will get an error in `serialize_parse_data_nested_helper()` since this column is required.